ui/components/policy/req.js

```diff
@@ -1,15 +1,64 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
+import Metadata from '../requirements/metadata';
+import PolicyLink from '../requirements/policy-link';
+import TopicLink from '../requirements/topic-link';
+
+const badEntities = [
+  'NA',
+  'N/A',
+  'Not Applicable',
+  'None',
+  'None Specified',
+  'unknown',
+  'TBA',
+  'Cannot determine-Ask Mindy',
+].map(e => e.toLowerCase());
+
+export function filterAppliesTo(text) {
+  const normalized = (text || '').toLowerCase().trim();
+  if (badEntities.includes(normalized)) {
+    return null;
+  }
+  return text;
+}
```

Comment:
What do you think about importing `filterAppliesTo` rather than copy-pasting it? We'll probably move logic around in the future, but it'd be better if we knew the function hadn't changed.

Reply:
Love this idea.